hand_bot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  index_tip_controller:
    type: position_controllers/JointPositionController
    joint: index_tip_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  index_middle_controller:
    type: position_controllers/JointPositionController
    joint: index_middle_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  index_lower_controller:
    type: position_controllers/JointPositionController
    joint: index_lower_joint
    pid:
      p: 1
      i: 10
      d: 1

  middle_tip_controller:
    type: position_controllers/JointPositionController
    joint: middle_tip_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  middle_middle_controller:
    type: position_controllers/JointPositionController
    joint: middle_middle_joint
    pid:
      p: 1
      i: 1
      d: 1

  middle_lower_controller:
    type: position_controllers/JointPositionController
    joint: middle_lower_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  ring_tip_controller:
    type: position_controllers/JointPositionController
    joint: ring_tip_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  ring_middle_controller:
    type: position_controllers/JointPositionController
    joint: ring_middle_joint
    pid:
      p: 1
      i: 1
      d: 1

  ring_lower_controller:
    type: position_controllers/JointPositionController
    joint: ring_lower_joint
    pid:
      p: 1
      i: 1
      d: 1

  little_tip_controller:
    type: position_controllers/JointPositionController
    joint: little_tip_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  little_middle_controller:
    type: position_controllers/JointPositionController
    joint: little_middle_joint
    pid:
      p: 1
      i: 1
      d: 1

  little_lower_controller:
    type: position_controllers/JointPositionController
    joint: little_lower_joint
    pid:
      p: 1
      i: 1
      d: 1

  thumb_tip_controller:
    type: position_controllers/JointPositionController
    joint: thumb_tip_joint
    pid:
      p: 1
      i: 1
      d: 1
      
  thumb_lower_controller:
    type: position_controllers/JointPositionController
    joint: thumb_lower_joint
    pid:
      p: 1
      i: 0.1
      d: 1
 
